apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -181,17 +185,17 @@ export class DetailViewContents extends React.Component {
   };
 
   handleRegionalPartnerChange = (selected) => {
-    const regional_partner_filter = selected ? selected.value : UnmatchedFilter;
-    const regional_partner_name = selected ? selected.label : UnmatchedLabel;
-    this.setState({ regional_partner_name, regional_partner_filter});
+    const regional_partner_id = selected ? selected.value : UnmatchedPartnerValue;
+    const regional_partner_name = selected ? selected.label : UnmatchedPartnerLabel;
+    this.setState({ regional_partner_name, regional_partner_id});
```

Comment:
This part doesn't appear to be new, so I assume it works, but what if `all` or `none` come through as selected?